run_test.sh

```diff
@@ -188,10 +189,7 @@ function update_integration_test_golden {
   export LLM_API_KEY=
   export ENABLE_EVAL_TOOL=true
 
-  export ENV_PREFIX=Autopush
-  python3 -m pytest -vv server/integration_tests/topic_cache
```

Comment:
Don't remove the test?

Reply:
this is already covered in the test below?

Reply:
oops, yes